test/integration/registered_user_cannot_view_anothers_users_dashboard_test.rb

```diff
@@ -0,0 +1,56 @@
+require 'test_helper'
+
+class RegisteredUserCannotViewAnothersUsersDashboardTest < ActionDispatch::IntegrationTest
+  include CategoryItemsSetup
+
+  test 'a registered user cannot view another users dashboard' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.